<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1245</width>
    <height>687</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="minimumSize">
   <size>
    <width>1245</width>
    <height>687</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>1245</width>
    <height>687</height>
   </size>
  </property>
  <property name="font">
   <font>
    <family>Segoe UI</family>
   </font>
  </property>
  <property name="windowTitle">
   <string>Products</string>
  </property>
  <property name="autoFillBackground">
   <bool>false</bool>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QLabel" name="labelProduct">
    <property name="geometry">
     <rect>
      <x>240</x>
      <y>90</y>
      <width>361</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>14</pointsize>
      <weight>75</weight>
      <italic>false</italic>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>Products</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEditSearch">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>30</y>
      <width>1081</width>
      <height>40</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>10</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">QLineEdit {
            border-radius: 10px;
            padding: 5px;
            border: 2px solid #ccc;
        }

        QLineEdit:focus {
            border: 2px solid #008080;
        }</string>
    </property>
    <property name="placeholderText">
     <string>Search Product</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushBtnSearch">
    <property name="geometry">
     <rect>
      <x>1130</x>
      <y>30</y>
      <width>75</width>
      <height>40</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>10</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton {
            border-radius: 10px;
            padding: 5px;
            background-color: #009688;
            color: white;
        }

QPushButton:hover {
            background-color: #008080;
        }

QPushButton:pressed {
            background-color: #006666;
        }</string>
    </property>
    <property name="text">
     <string>Search</string>
    </property>
   </widget>
   <widget class="QGroupBox" name="groupBoxFilter">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>100</y>
      <width>181</width>
      <height>541</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>12</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="title">
     <string>Filter</string>
    </property>
    <widget class="QLabel" name="labelPriceRange">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>30</y>
       <width>111</width>
       <height>31</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>11</pointsize>
       <weight>75</weight>
       <bold>true</bold>
      </font>
     </property>
     <property name="text">
      <string>Price Range</string>
     </property>
    </widget>
    <widget class="QLabel" name="labelWebsite">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>210</y>
       <width>111</width>
       <height>31</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>11</pointsize>
       <weight>75</weight>
       <bold>true</bold>
      </font>
     </property>
     <property name="text">
      <string>Website</string>
     </property>
    </widget>
    <widget class="QCheckBox" name="checkTarget">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>310</y>
       <width>101</width>
       <height>17</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>10</pointsize>
       <weight>50</weight>
       <bold>false</bold>
      </font>
     </property>
     <property name="text">
      <string>Target</string>
     </property>
    </widget>
    <widget class="QCheckBox" name="checkCostco">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>270</y>
       <width>101</width>
       <height>17</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>10</pointsize>
       <weight>50</weight>
       <bold>false</bold>
      </font>
     </property>
     <property name="text">
      <string>Costco</string>
     </property>
    </widget>
    <widget class="QCheckBox" name="checkIkea">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>290</y>
       <width>101</width>
       <height>17</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>10</pointsize>
       <weight>50</weight>
       <bold>false</bold>
      </font>
     </property>
     <property name="text">
      <string>Ikea</string>
     </property>
    </widget>
    <widget class="QCheckBox" name="checkWalmart">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>330</y>
       <width>131</width>
       <height>17</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>10</pointsize>
       <weight>50</weight>
       <bold>false</bold>
      </font>
     </property>
     <property name="text">
      <string>Walmart</string>
     </property>
    </widget>
    <widget class="QCheckBox" name="checkAmazon">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>250</y>
       <width>101</width>
       <height>17</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>10</pointsize>
       <weight>50</weight>
       <bold>false</bold>
      </font>
     </property>
     <property name="text">
      <string>Amazon</string>
     </property>
    </widget>
    <widget class="QRadioButton" name="radioAll">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>70</y>
       <width>82</width>
       <height>17</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>10</pointsize>
       <weight>50</weight>
       <bold>false</bold>
      </font>
     </property>
     <property name="text">
      <string>All</string>
     </property>
    </widget>
    <widget class="QRadioButton" name="radio25_100">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>110</y>
       <width>141</width>
       <height>17</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>10</pointsize>
       <weight>50</weight>
       <bold>false</bold>
      </font>
     </property>
     <property name="text">
      <string>$25 - $100</string>
     </property>
    </widget>
    <widget class="QRadioButton" name="radio100_500">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>130</y>
       <width>141</width>
       <height>17</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>10</pointsize>
       <weight>50</weight>
       <bold>false</bold>
      </font>
     </property>
     <property name="text">
      <string>$100 - $500</string>
     </property>
    </widget>
    <widget class="QRadioButton" name="radio500_1000">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>150</y>
       <width>141</width>
       <height>17</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>10</pointsize>
       <weight>50</weight>
       <bold>false</bold>
      </font>
     </property>
     <property name="text">
      <string>$500 - $1000</string>
     </property>
    </widget>
    <widget class="QRadioButton" name="radio1000">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>170</y>
       <width>161</width>
       <height>17</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>10</pointsize>
       <weight>50</weight>
       <bold>false</bold>
      </font>
     </property>
     <property name="text">
      <string>$1000 and Above</string>
     </property>
    </widget>
    <widget class="QRadioButton" name="radio0_25">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>90</y>
       <width>141</width>
       <height>17</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>10</pointsize>
       <weight>50</weight>
       <bold>false</bold>
      </font>
     </property>
     <property name="text">
      <string>$0 - $25</string>
     </property>
    </widget>
    <widget class="QComboBox" name="comboBoxCategory">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>410</y>
       <width>131</width>
       <height>22</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>11</pointsize>
       <weight>50</weight>
       <bold>false</bold>
      </font>
     </property>
     <item>
      <property name="text">
       <string>All</string>
      </property>
     </item>
    </widget>
    <widget class="QLabel" name="labelCategory">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>370</y>
       <width>111</width>
       <height>31</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>11</pointsize>
       <weight>75</weight>
       <bold>true</bold>
      </font>
     </property>
     <property name="text">
      <string>Category</string>
     </property>
    </widget>
    <widget class="QPushButton" name="pushBtnApplyFilter">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>450</y>
       <width>121</width>
       <height>40</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>10</pointsize>
       <weight>75</weight>
       <bold>true</bold>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">QPushButton {
            border-radius: 10px;
            padding: 5px;
            background-color: #009688;
            color: white;
        }

QPushButton:hover {
            background-color: #008080;
        }

QPushButton:pressed {
            background-color: #006666;
        }</string>
     </property>
     <property name="text">
      <string>Apply Filter</string>
     </property>
    </widget>
   </widget>
   <widget class="QTableWidget" name="tableWidgetProduct">
    <property name="geometry">
     <rect>
      <x>240</x>
      <y>130</y>
      <width>971</width>
      <height>521</height>
     </rect>
    </property>
    <property name="verticalScrollBarPolicy">
     <enum>Qt::ScrollBarAlwaysOn</enum>
    </property>
    <property name="horizontalScrollBarPolicy">
     <enum>Qt::ScrollBarAlwaysOff</enum>
    </property>
    <property name="sizeAdjustPolicy">
     <enum>QAbstractScrollArea::AdjustToContentsOnFirstShow</enum>
    </property>
    <property name="editTriggers">
     <set>QAbstractItemView::NoEditTriggers</set>
    </property>
    <property name="rowCount">
     <number>0</number>
    </property>
    <property name="columnCount">
     <number>6</number>
    </property>
    <attribute name="horizontalHeaderCascadingSectionResizes">
     <bool>true</bool>
    </attribute>
    <attribute name="horizontalHeaderDefaultSectionSize">
     <number>161</number>
    </attribute>
    <attribute name="verticalHeaderVisible">
     <bool>true</bool>
    </attribute>
    <column>
     <property name="text">
      <string>Product Name</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>Category</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>Product Price</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>Website Name</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>Quantity</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>Action</string>
     </property>
    </column>
   </widget>
   <widget class="QPushButton" name="pushBtnMyCart">
    <property name="geometry">
     <rect>
      <x>1130</x>
      <y>90</y>
      <width>75</width>
      <height>30</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>8</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton {
            border-radius: 6px;
            padding: 5px;
            background-color: #DC143C;
            color: white;
        }

QPushButton:hover {
            background-color: #DA012D;
        }

QPushButton:pressed {
            background-color: #006666;
        }</string>
    </property>
    <property name="text">
     <string>My Cart</string>
    </property>
   </widget>
   <widget class="QLabel" name="labelSearchMessage">
    <property name="geometry">
     <rect>
      <x>520</x>
      <y>380</y>
      <width>251</width>
      <height>41</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>12</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QPushButton" name="pushBtnMyOrder">
    <property name="geometry">
     <rect>
      <x>1014</x>
      <y>90</y>
      <width>91</width>
      <height>30</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>8</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton {
            border-radius: 6px;
            padding: 5px;
            background-color: #DC143C;
            color: white;
        }

QPushButton:hover {
            background-color: #DA012D;
        }

QPushButton:pressed {
            background-color: #006666;
        }</string>
    </property>
    <property name="text">
     <string>My Orders</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1245</width>
     <height>26</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
